Cargo.toml

```diff
@@ -57,7 +57,8 @@ unused_extern_crates = "deny"
 
 [workspace.lints.clippy]
 type_complexity = "allow"
-all = "deny"
+# Priority -1 means that it can overwritten by other lints.
```

Comment:
```suggestion
# Priority -1 means that it can overwritten by other lints, https://rust-lang.github.io/rust-clippy/master/index.html#/lint_groups_priority
```